docs/docs/user-guide/yaml-config.md

```diff
@@ -0,0 +1,48 @@
+# YAML Crawl Config
+
+Browsertix Crawler supports the use of a yaml file to set parameters for a crawl. This can be used by passing a valid yaml file to the `--config` option.
```

Comment:
Capitalized below
```suggestion
Browsertix Crawler supports the use of a YAML file to set parameters for a crawl. This can be used by passing a valid yaml file to the `--config` option.
```